test/catch.hpp

```diff
@@ -10280,7 +10280,7 @@ namespace Catch {
 #ifndef __OBJC__
 
 // Standard C/C++ main entry point
-int main (int argc, char * argv[]) {
+int main (int argc, const char * argv[]) {
```

Comment:
This whole file is copied verbatim from the `catch` distribution and so is likely to be overwritten. You can probably leave this one out.

Reply:
You're right, I went a bit too happy with Find+Replace!